docker-compose.yml

```diff
@@ -96,6 +96,8 @@ services:
     image: nasa-apt/dev/app
     command: >
       sh -c "
+        python3 -m pip install types-requests &&
```

Comment:
Should these two libraries go into the `setup.py`?

Reply:
I *think* this can be removed now. I had added this to try and resolve the same issue where `types-requests` wasn't passing the mypy linter.

Reply:
This should stay to ease any typing issues preferably without ignore statements.